src/network/networkd-route.c

```diff
@@ -2573,6 +2573,17 @@ static int route_section_verify(Route *route, Network *network) {
                 route->gateway_onlink = true;
         }
 
+        if (route->family == AF_INET6) {
```

Comment:
is the other way around possible and worth checking as well? INET route configured with INET6 multipath route

Reply:
Good catch. But interestingly, IPv4 route with IPv6 multipath route is certainly supported by kernel and ip tools.

Reply:
oh, interesting